Moodlet/app/src/androidTest/java/com/cmput3owo1/moodlet/activities/RegisterErrorTests.java

```diff
@@ -0,0 +1,122 @@
+package com.cmput3owo1.moodlet.activities;
```

Comment:
Minor thing: Change file to just be "Test" not "Tests". Otherwise LGTM.

Reply:
fixed!